components/ui/Gnb.vue

```diff
@@ -100,6 +110,14 @@ export default {
             } else {
                 this.$router.push('/' + key + '/' + this.currentSeason);
             }
+            let seasonInfo = this.$store.state.seasonList.filter(res => {
```

Comment:
Codacy has a fix for the issue: [Replace `(res` with `((res)`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
            let seasonInfo = this.$store.state.seasonList.filter((res) => {
```